docs/kratos/organizations/organizations.mdx

```diff
@@ -0,0 +1,117 @@
+---
+id: organizations
+title: Organizations and B2B Single Sign-On
+---
+
+```mdx-code-block
+import Mermaid from "@site/src/theme/Mermaid"
+```
+
+## Introduction
```

Comment:
```suggestion
```

IMO the introductory text is enough; and best practice is not to put headlines after each other with no text between.